File: sample/sample/PlacementsViewController.m
//
//  PlacementsViewController.m
//  sample
//
//  Created by Vladas Drejeris on 04/05/15.
//  Copyright (c) 2015 Adform. All rights reserved.
//

#import "PlacementsViewController.h"
#import "AdhesionViewController.h"

@interface PlacementsViewController ()

@end

@implementation PlacementsViewController

- (void)viewDidLoad {
    [super viewDidLoad];
}

- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

#pragma mark - Navigation

- (void)prepareForSegue:(UIStoryboardSegue *)segue sender:(id)sender {

    if ([segue.identifier isEqualToString:@"AdhesionTop"]) {
        [(AdhesionViewController *)segue.destinationViewController setAdPosition:AFAdPositionTop];
    } else if ([segue.identifier isEqualToString:@"AdhesionBottom"]) {
        [(AdhesionViewController *)segue.destinationViewController setAdPosition:AFAdPositionBottom];
    }
}

@end

File: sample/sample/PlacementsViewController.swift
//
//  ViewController.swift
//  sample
//
//  Created by Laurynas Letkauskas on 2021-02-25.
//

import UIKit
import AdformAdvertising

class PlacementsViewController: UITableViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
    }
    
    override func prepare(for segue: UIStoryboardSegue, sender: Any?) {
        if segue.identifier == "AdhesionTop" {
            (segue.destination as? AdhesionViewController)?.adPosition = .top
        } else if segue.identifier == "AdhesionBottom" {
            (segue.destination as? AdhesionViewController)?.adPosition = .bottom
        }
    }

}

